src/templates/en/2020/stories/content_distribution.html

```diff
@@ -0,0 +1,1767 @@
+              <div id="el-cdb28853-0908-4d31-a1ed-10a6fb4954f4" data-amp-original-style="pointer-events:initial;width:100%;height:100%;display:block;position:absolute;top:0;left:0" class="amp-wp-a1b70b8"><p class="fill amp-wp-ae8dc76" data-amp-original-style="white-space:pre-wrap;overflow-wrap:break-word;word-break:break-word;margin:-1.325088339222615% 0;font-family:'Lato',sans-serif;font-size:0.404531em;line-height:1.5;text-align:initial;padding:0% 0%;color:#000000"><span data-amp-original-style="color: #fcfcfc" class="amp-wp-4a61a81">HTTP Archive's annual</span><br><span data-amp-original-style="font-weight: 900; color: #fcfcfc" class="amp-wp-f94bb8b">state of the web</span><span data-amp-original-style="color: #fcfcfc" class="amp-wp-4a61a81"> report</span></p></div>
+            </div>
+            <div data-amp-original-style="position:absolute;pointer-events:none;left:-3.8835%;top:63.26861%;width:134.70874%;height:41.26214%;transform:rotate(354deg);opacity:1" class="amp-wp-9fa0714">
+              <div class="mask amp-wp-a1b70b8" id="el-90c94fbb-06eb-4862-b819-0f046d5ab371" data-amp-original-style="pointer-events:initial;width:100%;height:100%;display:block;position:absolute;top:0;left:0">
+                <div class="fill amp-wp-a511151" data-amp-original-style="background-color:#5b667a"></div>
+              </div>
+            </div>
+            <div data-amp-original-style="position:absolute;pointer-events:none;left:-7.76699%;top:67.79935%;width:134.70874%;height:35.92233%;transform:rotate(354deg);opacity:1" class="amp-wp-e2aa3e8">
+              <div class="mask amp-wp-a1b70b8" id="el-d9fba4a5-b10e-4b24-9f82-2f3992c2bf23" data-amp-original-style="pointer-events:initial;width:100%;height:100%;display:block;position:absolute;top:0;left:0">
+                <div class="fill amp-wp-b3b6893" data-amp-original-style="background-color:#545f74"></div>
+              </div>
+            </div>
+            <div data-amp-original-style="position:absolute;pointer-events:none;left:9.95146%;top:79.61165%;width:80.09709%;height:43.85113%;opacity:1" class="amp-wp-e8247a0">
+              <div class="mask amp-wp-a1b70b8" id="el-a2577370-d911-435c-a21d-826228c08231" data-amp-original-style="pointer-events:initial;width:100%;height:100%;display:block;position:absolute;top:0;left:0">
+                <div data-leaf-element="true" data-amp-original-style="position:absolute;width:100%;height:100.08762%;left:0%;top:-0.04381%" class="amp-wp-3fc8e8f">
+                  <amp-img layout="fill" src="/static/stories/web-almanac-2020-content-distribution/WA_Chat-Window.png" alt="A graphic showing an example website" srcset="/static/stories/web-almanac-2020-content-distribution/WA_Chat-Window.png 994w, assets/WA_Chat-Window-768x631.png 768w, assets/WA_Chat-Window-300x247.png 300w, assets/WA_Chat-Window-150x123.png 150w" class="i-amphtml-layout-fill i-amphtml-layout-size-defined" i-amphtml-layout="fill"></amp-img>
```

Comment:
We're missing some of the paths in the images - except for first one:

```suggestion
                  <amp-img layout="fill" src="/static/stories/web-almanac-2020-content-distribution/WA_Chat-Window.png" alt="A graphic showing an example website" srcset="/static/stories/web-almanac-2020-content-distribution/WA_Chat-Window.png 994w, /static/stories/web-almanac-2020-content-distribution/WA_Chat-Window-768x631.png 768w, /static/stories/web-almanac-2020-content-distribution/WA_Chat-Window-300x247.png 300w, /static/stories/web-almanac-2020-content-distribution/WA_Chat-Window-150x123.png 150w" class="i-amphtml-layout-fill i-amphtml-layout-size-defined" i-amphtml-layout="fill"></amp-img>
```

More than this example.

Reply:
Fixed it with a quick search and replace and looks a better now!

Reply:
Thanks @bazzadp! Not sure  how my search+replace missed those instances.